code/modules/projectiles/guns/rifles.dm

```diff
@@ -408,7 +408,7 @@
 							/obj/item/attachable/heavy_barrel,
 								)
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER
-	unacidable = TRUE
+	//unacidable = TRUE
```

Comment:
remove instead of commenting out

commenting out stuff like this is bad practice

Reply:
I thought the opposite was the case lol, no problem.